modules/tinydnsbackend/tinydnsbackend.hh

```diff
@@ -73,7 +73,7 @@ private:
 	typedef TDI_t::index<tag_domainid>::type TDIById_t;
 
 	//data member variables
-	uint64_t d_taiepoch;
+	unsigned long long d_taiepoch;
```

Comment:
This can't be a separate pull request.
Cassandra needs upgrade of libc6 and libc6-dev, this causes failure to tinydns due to namespace conflict between std::uint64_t and boost::uint64_t